parlai/scripts/convo_render.py

```diff
@@ -209,6 +257,17 @@ def display_cli(conversations, alt_speaker, human_speaker):
                 if extension == "pdf":
                     cmd = f"{CHROME_PATH} --headless --crash-dumps-dir=/tmp --print-to-pdf=\"{output_file}\" {fname}"
                 else:
-                    cmd = f"{CHROME_PATH} --headless --crash-dumps-dir=/tmp --screenshot=\"{output_file}\" {fname}"
-                os.system(cmd)
+                    if len(args.window_size) != 2:
+                        raise Exception("Invalid window size provided")
+                    if (
+                        not args.window_size[0].isdigit()
+                        or not args.window_size[1].isdigit()
+                    ):
+                        raise ValueError(
+                            "Please provide integer values for window size"
+                        )
+                    cmd = f"{CHROME_PATH} --headless --crash-dumps-dir=/tmp --window-size={args.window_size[0]},{args.window_size[1]} --screenshot=\"{output_file}\" {fname}"
```

Comment:
wrap multiple lines

Reply:
Could you elaborate a bit on that please?

Reply:
Line length should be under 88 characters. 

```python
cmd = (
    f"{CHROME_PATH} --headless --crash-dumps-dir=/tmp "
    f"--window-size={args.window_size[0]},{args.window_size[1]} "
    f"--screenshot=\"{output_file}\" {fname}"
)
```